deploy/kubernetes/operator.nsx.vmware.com_ncpinstalls_crd.yaml

```diff
@@ -37,6 +37,9 @@ spec:
               type: integer
               format: int32
               minimum: 1
+            addNodeTag:
+              description: Skip tagging node logical switch ports with node name and cluster
```

Comment:
Probably the description should something like "Tag node logical switch port ... when set to True, skip tagging when set to False". In the way it is written now, it seems we need to set it to True to skip tagging

Reply:
Yes, I changed the option from "disableTag" to "addNodeTag" but forgot to modify the description.. Thanks!